client/memez/projects/memez/src/app/stores/entities/user.store.ts

```diff
@@ -1,42 +1,42 @@
 import {Injectable}         from '@angular/core';
 import {action, observable} from 'mobx-angular';
 import {RootStore}          from '../root.store';
-import {IUser}              from '../../types/Entities/IUser';
+import {IUser}              from '../../../../../../../../shared/types/Entities/IUser';
+import {autorun, reaction}  from 'mobx';
+/*import {MOCK_USERS}         from '../../../../../../../../shared/mock/MOCK_USERS';*/
+import {ILike}              from '../../../../../../../../shared/types/Entities/ILike';
 
-import {autorun}    from 'mobx';
-import {MOCK_USERS} from '../../../../../../../../shared/mock/MOCK_USERS';
-import {ILike}      from '../../types/Entities/ILike';
 
 @Injectable({
   providedIn: 'root'
 })
 export class UserStore {
-  useMock: boolean                  = true;
+  /*useMock: boolean = false;*/
   @observable public users: IUser[] = [];
 
   constructor(
     public root: RootStore
   ) {
-    this.root.us        = this;
+    this.root.us = this;
     window['userStore'] = this;
-    autorun(() => {
-      console.log(`users array has changed to ${this.users.map(user => user.name)}`);
-    });
+
   }
 
   @action
   async getUsers(): Promise<IUser[]> {
-    if (this.useMock) {
+    /*if (this.useMock) {
       this.users = MOCK_USERS;
       return this.users;
-    }
+    }*/
 
     return this.users = await this.root
```

Comment:
I just realised that your root store depends on all of the adapters. I think it would be cleaner to inject the adapter only to the stores that need it